src/Tizen.NUI/src/public/BaseHandle.cs

```diff
@@ -559,6 +566,22 @@ public class FocusRequestArgs : EventArgs
                 swigCPtr = value;
             }
         }
+
+        /// <summary>
+        /// swigCMemOwn
+        /// </summary>
+        internal protected bool SwigCMemOwn
+        {
+            get => swigCMemOwn;
+        }
+
+        /// <summary>
+        /// A flag to check if it is already disposed.
+        /// </summary>
+        internal protected bool Disposed
+        {
+            get => disposed;
+        }
```

Comment:
You can use expression body definition to implement a read-only property.
(https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members#read-only-properties)
```suggestion
        internal protected bool Disposed => disposed;
```

Reply:
done